File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/java/F11.P1.java
import java.util.ArrayList;
import java.util.List;

class Main {
    static int element(int i) {
        var xs = new ArrayList<>(List.of(1, 2, 3));
        // BEGIN PROBE F11.P1
        int e = xs.get(i);
        return e;
        // END PROBE F11.P1
    }

    public static void main(String[] args) {
        System.out.println(element(2));
    }
}

File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/cpp/F11.P1.cpp
#include <cstdio>
#include <vector>

int element(int i) {
    std::vector<int> xs{1, 2, 3};
// BEGIN PROBE F11.P1
    int e = xs[i];
    return e;
// END PROBE F11.P1
}

int main() {
    std::printf("%d\n", element(2));
}
